scripts/linters/pylint_extensions.py

```diff
@@ -47,7 +47,7 @@
 ]
 
 ALLOWED_PRAGMAS_FOR_INLINE_COMMENTS: Final = [
-    'pylint:', 'isort:', 'type: ignore', 'pragma:', 'https:'
+    'pylint:', 'isort:', 'type: ignore', 'pragma:', 'https:', 'docker'
```

Comment:
```suggestion
    'pylint:', 'isort:', 'type: ignore', 'pragma:', 'https:', 'docker:'
```

to match the style of the other pragmas